### DISCUSSION TITLE
New environment variable for nuget.config

### DISCUSSION THREAD

**Discussion by michha** [renovatebot/renovate#24481] [Suggest an Idea]
### Tell us more.

My setting:
- company with 100+ applications, each in custom repo
- custom private nuget feeds
- no `nuget.config` inside repos
- top level `nuget.config` with feed and source package mapping (I am aware of #17562)

I would like to add a new environment variable (e.g. `NUGET_CONFIG`) to renovate and test for it in [util.ts](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/nuget/util.ts#L35).
If set, renovate would take this path instead of looking for a nuget config in the local repository.

**Comment by rarkins** [comment]
Do you mean that you want to add a `nuget.config` file to a location like `/home/ubuntu` within the container, and have Renovate find that?

**Reply by michha** [reply]
exactly

**Reply by rarkins** [reply]
There's already an existing concept for this, which is currently supported by npm only. I've created a feature request issue for it in nuget: https://github.com/renovatebot/renovate/issues/24485

PRs welcome, if you're so inclined
